pages/api/add-players.js

```diff
@@ -0,0 +1,72 @@
+import clientPromise from "../../lib/mongodb";
+
+import initMiddleware from "../../lib/init-middleware";
+import validateMiddleware from "../../lib/validate-middleware";
+import { check, validationResult } from "express-validator";
+
+const validateBody = initMiddleware(
```

Comment:
Using `initMiddleware` seems pretty nice and convey the same experience of express.js's `middleware` pattern. 

But it makes it a bit more complex. If we want to simplify, we can just delete `initMiddleware` here and then, in `validateMiddleware` just do `return` where `next()` is called currently. So it can continue.